docs/data/material/components/app-bar/app-bar.md

```diff
@@ -154,7 +154,7 @@ You can override this behavior by setting the `enableColorOnDark` prop to `true`
 
 {{"demo": "EnableColorOnDarkAppBar.js", "bg": true}}
 
-## Experimental APIs
+## Toolpad
```

Comment:
It's not stable: https://www.npmjs.com/package/@toolpad/core?activeTab=versions?

```suggestion
## Toolpad (alpha)
```

x the other headers.